var/spack/repos/builtin/packages/py-mdanalysis/package.py

```diff
@@ -18,141 +18,52 @@ class PyMdanalysis(PythonPackage):
-        deprecated=True,
-    )
-    version(
-        "0.16.0",
-        sha256="c4824fa1fddd336daa39371436187ebb023366885fb250c2827ed7fce2546bd4",
-        deprecated=True,
-    )
-    version(
-        "0.15.0",
-        sha256="9088786048b47339cba1f8a586977bbb3bb04ae1bcd0462b59e45bda37e25533",
-        deprecated=True,
-    )
 
     variant(
         "analysis",
         default=True,
         description="Enable analysis packages: matplotlib, scipy, seaborn",
     )
-    variant("amber", default=False, when="@:1.0.0", description="Support AMBER netcdf format.")
-    variant("extra_formats", default=False, when="@2.4.0:", description="Support extra formats")
+    variant("extra_formats", default=False, description="Support extra formats")
 
-    depends_on("python@2.7:", type=("build", "run"), when="@:1.0.0")
-    depends_on("python@3.8:", type=("build", "run"), when="@2.4.0:")
+    depends_on("python@3.9:", type=("build", "run"), when="@2.5.0:")
+    depends_on("python@3.8:", type=("build", "run"))
 
     depends_on("py-setuptools", type="build")
 
-    depends_on("py-cython@0.16:", type="build", when="@:1.0.0")
-    depends_on("py-cython@0.28:", type="build", when="@2.4.0:")
+    depends_on("py-cython@0.28:", type="build")
 
     # MDAnalysis required dependencies (install_requires)
-    depends_on("py-numpy@1.5.0:", when="@:0.15.0", type=("build", "run"))
-    depends_on("py-numpy@1.10.4:", when="@0.16.0:0.19.2", type=("build", "run"))
-    depends_on("py-numpy@1.13.3:", when="@0.20.1:1.0.0", type=("build", "run"))
-    depends_on("py-numpy@1.20.0:", when="@2.4.0:", type=("build", "run"))
-
-    depends_on("py-biopython@1.59:", when="@:0.17.0", type=("build", "run"))
-    depends_on("py-biopython@1.71:", when="@0.18.0:1.0.0", type=("build", "run"))
-    depends_on("py-biopython@1.80:", when="@2.4.0:", type=("build", "run"))
-
-    depends_on("py-networkx@1.0:", type=("build", "run"), when="@:1.0.0")
-    depends_on("py-networkx@2.0:", type=("build", "run"), when="@2.4.0:")
-
-    depends_on("py-griddataformats@0.3.2:", when="@:0.16.2", type=("build", "run"))
-    depends_on("py-griddataformats@0.4.0:", when="@0.17.0:", type=("build", "run"))
+    depends_on("py-numpy@1.21.0:", when="@2.5.0:", type=("build", "run"))
+    depends_on("py-numpy@1.20.0:", type=("build", "run"))
 
-    depends_on("py-mmtf-python@1.0.0:", when="@0.16.0:", type=("build", "run"))
+    depends_on("py-biopython@1.80:", type=("build", "run"))
+    depends_on("py-networkx@2.0:", type=("build", "run"))
+    depends_on("py-griddataformats@0.4.0:", type=("build", "run"))
+    depends_on("py-mmtf-python@1.0.0:", type=("build", "run"))
+    depends_on("py-joblib@0.12:", type=("build", "run"))
 
-    depends_on("py-joblib", when="@0.16.0:0.20.1", type=("build", "run"))
-    depends_on("py-joblib@0.12:", when="@1.0.0:", type=("build", "run"))
+    depends_on("py-scipy@1.5.1:", when="@2.5.0:", type=("build", "run"))
```

Comment:
It's still 1.5.0 in 2.5.0 isn't it?

Reply:
Yep, I think I might have confused it with matplotlib just below.